### PR TITLE
Us006 add envs

### PR DESCRIPTION
**Pull request by benjefferies**
Adding missing configuration to docker compose files.

To test run `make up` and do a sanity check

### CODE REVIEW

**Comment by insacuri** [comment]
Need to fix the conflict

File: ras-services.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ services:
       - RAS_OAUTH_CLIENT_ID=ons@ons.gov
       - RAS_OAUTH_CLIENT_SECRET=password
       - RAS_CASE_SERVICE_HOST=casesvc
-      - RAS_COLLEX_SERVICE_HOST=collex
+      - RAS_COLLEX_SERVICE_HOST=${COLL_INST_HOST}
```
Comment (andrewmil) [inline ras-services.yml]: I assume COLLEX should mean collection exercise rather than instrument?

File: ras-services.yml [hunk 2]
```diff
@@ -112,7 +112,7 @@ services:
       -  RAS_PARTY_SERVICE_HOST=party-service
       -  RAS_PARTY_SERVICE_PORT=8081
       -  RAS_PARTY_SERVICE_PROTOCOL=http
-      -  RM_COLLECTION_EXERCISE_SERVICE_HOST=collex
+      -  RM_COLLECTION_EXERCISE_SERVICE_HOST=${COLL_INST_HOST}
```
Comment (andrewmil) [inline ras-services.yml]: Collection exercise != instrument

File: ras-services.yml [hunk 3]
```diff
@@ -85,7 +85,7 @@ services:
       - "${FRONTSTAGE_PORT}:${FRONTSTAGE_PORT}"
     environment:
       -  API_GATEWAY_CASE_URL=http://casesvc:8171/cases/
-      -  API_GATEWAY_COLLECTION_EXERCISE_URL=http://collex:8145/collectionexercises/
+      -  API_GATEWAY_COLLECTION_EXERCISE_URL=http://${COLL_INST_HOST}:8145/collectionexercises/
```
Comment (andrewmil) [inline ras-services.yml]: Collection exercise != instrument

File: ras-services.yml [hunk 4]
```diff
@@ -135,17 +135,19 @@ services:
     container_name: collection-instrument
     image: sdcplatform/ras-collection-instrument
     ports:
-      - "${COLLECTION_INSTRUMENT_PORT}:${COLLECTION_INSTRUMENT_PORT}"
+      - "${COLL_INST_PORT}:${COLL_INST_PORT}"
     environment:
       - DATABASE_URI=postgresql://${POSTGRES_USERNAME}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/postgres
       - API_GATEWAY_CASE_URL=http://casesvc:8171/cases/
-      - API_GATEWAY_COLLECTION_EXERCISE_URL=http://collex:8145/collectionexercises/
+      - API_GATEWAY_COLLECTION_EXERCISE_URL=http://${COLL_INST_HOST}:8145/collectionexercises/
```
Comment (andrewmil) [inline ras-services.yml]: Collection exercise != instrument

Comment (benjefferies) [reply ras-services.yml]: Fixed. Good spot

**Pull request merged**